libs/components/docs-tools/src/lib/modules/code-snippet/code-snippet.component.scss

```diff
@@ -0,0 +1,3 @@
+sky-code-snippet {
```

Comment:
Need to disable view encapsulation so that we can style the generated `span` elements from `highlight.js`.